shopify_features/models/profiles.yaml

```diff
@@ -578,28 +602,33 @@ models:
               order_by: 
               - case when TOTAL_PRICE_USD is not null then 2 else 1 end desc
               - timestamp desc
+            description: The total value of products that are part of the last transaction.
 
         
         - entity_var:
             name: total_products_added
             select: array_agg(distinct product_id)
             from: models/rsCartLineItems
-
+            description: Total products added till date. (array with list of all product ids)
```

Comment:
Modify to:
            description: Total products added to cart till date. (array with list of all product ids); It includes all purchased products plus current active cart. 

Reuse this format for products_added_in_past_n_days features.